packages/pn-commons/src/components/NotificationDetail/__test__/NotificationPaymentRecipient.test.tsx

```diff
@@ -21,6 +22,14 @@ describe('NotificationPaymentRecipient Component', () => {
     f24Only: getF24Payments(payments, 0),
   };
 
+  beforeAll(() => {
```

Comment:
is this useful for this test suite?

Reply:
I can move it into pagination test (should call handleFetchPaymentsInfo on pagination click) because i have to set that properties in order to pass the validation of sessionStorage. 
In my opinion can be safe also to kept into beforeAll

Reply:
it's fine how you did it